src/engine/server/server.cpp

```diff
@@ -1424,10 +1423,12 @@ int CServer::Run()
 				}
 			}
 
-			while(t > TickStartTime(m_CurrentGameTick+1))
+			int64 Now = time_get();
+			bool NewTicks = false;
+			while(Now > TickStartTime(m_CurrentGameTick+1))
 			{
 				m_CurrentGameTick++;
-				NewTicks++;
```

Comment:
Should probably detect if we skip a tick here. Cause when it jumps from an odd`m_CurrentGameTick` to another one it'll skip snapping the game some lines below